yb-voyager/src/srcdb/postgres.go

```diff
@@ -501,7 +501,13 @@ WHERE parent.relname='%s' AND nmsp_parent.nspname = '%s' `, tableName.ObjectName
 		if err != nil {
 			utils.ErrExit("Error in scanning for child partitions of table=%s: %v", tableName, err)
 		}
-		partitions = append(partitions, sqlname.NewSourceName(childSchema, childTable))
+		if tableName.ObjectName.MinQuoted != tableName.ObjectName.Unquoted {
```

Comment:
Is this a reliable way to figure out if a name is case-sensitive? I'm not entirely clear on what MinQuoted is.

Reply:
Yes, the `MinQuoted` means that the table name will be quoted if required which means for PG/YB in case of case-sensitive table name, in `MinQuoted` it will be quoted.